{{cookiecutter.github_repository_name}}/src/{{cookiecutter.module_name}}/models.py

```diff
@@ -1,25 +1,35 @@
+import datetime
 import uuid
 
 from django.db import models
 
 
 class DateModel(models.Model):
-    updated = models.DateTimeField(auto_now=True)
     created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(auto_now=True)
 
     class Meta:
         abstract = True
 
     def __str__(self):
         return str(self.created)
 
+    def save(self, *args, **kwargs):
+        if not self.id:  # On create
+            self.created = datetime.datetime.now(tz=utc)
+
+        self.updated = datetime.datetime.now(tz=utc)
+        return super(DateModel, self).save(*args, **kwargs)
+
 
 class Company(DateModel):
     identifier = models.CharField(max_length=100, unique=True, db_index=True)
     admin = models.OneToOneField('{{cookiecutter.module_name}}.User',
         related_name='admin_company')
     secret = models.UUIDField()
     name = models.CharField(max_length=100, null=True, blank=True)
+    email = models.EmailField(null=True, blank=True)
```

Comment:
Don't think this is required. Think it is related to the notification service. So you can remove `email`.